README.md

```diff
@@ -90,6 +91,7 @@
 
 - [theheadless.dev](https://www.checklyhq.com/learn/headless/) - Practical guides and runnable examples on Playwright (and Puppeteer).
 - [playwrightsolutions.com](https://playwrightsolutions.com) - Curated Selection of Playwright Automated Test Problems and Solutions.
+- [serenity-js.org](https://serenity-js.org/handbook/web-testing/your-first-web-scenario/) - Learn to write high-quality, business-focused automated acceptance tests using Playwright and Serenity/JS.
```

Comment:
high-quality and business-focused seems very opinionated. Should We maybe rephrase it?

Like: Learn to write tests using the Serenity/JS framework with the Screenplay Pattern.

Reply:
Thanks @mxschmitt, I rephrased it. Let me know if you're happy with the latest version.